core/src/smartcontracts/isi/world.rs

```diff
@@ -358,13 +355,13 @@ pub mod isi {
             let raw_validator = self.object;
             let engine = wsv.engine.clone(); // Cloning engine is cheap
 
-            let (new_validator /*, new_permission_token_definitions*/) =
+            let (new_validator, new_permission_token_definitions) =
                 || -> Result<_, crate::smartcontracts::wasm::error::Error> {
```

Comment:
why is this a closure?

Reply:
Because there are two places with errors inside. And I should convert this errors with the same way. So this closure is more like `try-catch`